@kiva/kv-stream/README.md

```diff
@@ -0,0 +1,45 @@
+# kv-stream
```

Comment:
I wonder if a name like `kv-activity-feed` might be more accurate, to match the BE abstraction.